.cloudbuild/library_generation/library_generation.Dockerfile

```diff
@@ -40,9 +63,24 @@ ENV HOME=/home
 ENV OS_ARCHITECTURE="linux-x86_64"
 
 # install OS tools
-RUN apt-get update && apt-get install -y \
-	unzip openjdk-17-jdk rsync maven jq \
-	&& apt-get clean
+RUN apk update && apk add unzip curl rsync maven jq bash nodejs npm git
```

Comment:
Do we need `maven` in the final image?

As for java formatter, we've changed to download the jar so Java runtime is suffice.

Reply:
This is because there is no candidate in APK to download the runtime directly (`apk list jdk`). I'll double check this.

Reply:
Oh, well, `apk list` doesn't support fuzzy search. I found `openjdk11` by searching that exact string. Thanks for checking on this.